signbank/static/js/gloss_edit.js

```diff
@@ -137,13 +137,15 @@ $.editable.addInputType('multicheckbox', {
             return submitdata;
         };
 
-        $.each(settings.data || [], function(value, label) {
-            if (value === "selected") {
-                return;
-            }
-
-            $el.append($("<label><input type='checkbox' value='" + value + "' /></label><br />"));
-        });
+        // Output multicheckbox in alphabetical order of labels
+        Object.entries(settings.data)
```

Comment:
nit: the previous code felt it was possible for `data` to be undefined - I have no idea if that's true, but might be safer to include that:

```suggestion
        Object.entries(settings.data ?? [])
```

(then again, its unlikely we'll ever come back to change this, so leaving it out would ensure we will find out for sure if the fallback is actually needed 😅)

Reply:
Now you've got me wondering ...

Reply:
... yeah heck with it, let's do it. I don't _think_ `settings.data` can be undefined, but I like the idea that if it's empty nothing shows up rather than an exception is thrown.